### PR TITLE
Reinforce

### PR DESCRIPTION
**Pull request by jaimepedretp**
Merge all commits

### CODE REVIEW

**Review by xeviknal — COMMENTED** [review]
WTF! What about this merges in the middle of the branch!!! AAAARRRGG!

File: main.py [hunk 1]
```diff
@@ -3,13 +3,14 @@
 from environment import CarRacingEnv
 from trainer import Trainer
 
-# if gpu is to be used
+# if gpu is to be used. unfortunately my linux doesn't have drivers for that.
 device = torch.device("cuda") if False else torch.device("cpu")
 
+# Main loop
```
Comment (ziritrion) [inline main.py]: this is a bullshit comment, who the fuck put it in there.

**Review by eldarsilver — APPROVED** [review]
fucking awsome

**Review by ziritrion — CHANGES_REQUESTED** [review]
I demand this comment to be erased.

File: main.py [hunk 2]
```diff
@@ -3,13 +3,14 @@
 from environment import CarRacingEnv
 from trainer import Trainer
 
-# if gpu is to be used
+# if gpu is to be used. unfortunately my linux doesn't have drivers for that.
```
Comment (xeviknal) [inline main.py:6]: First comment

File: main.py (hunk 1 above)
Comment (xeviknal) [inline main.py]: second commit

File: main.py [hunk 3]
```diff
@@ -3,13 +3,14 @@
...
 device = torch.device("cuda") if False else torch.device("cpu")
 
+# Main loop
 if __name__ == "__main__":
     hyperparams = {
-        'num_episodes': 1250,  # Number of training episodes
-        'lr': 1e-2,  # Learning rate
+        'num_episodes': 300000,  # Number of training episodes
```
Comment (xeviknal) [inline main.py:11]: third commit

File: main.py [hunk 4]
```diff
@@ -3,13 +3,14 @@
...
+        'lr': 1e-9,  # Learning rate
```
Comment (xeviknal) [inline main.py:12]: asdasdasd

**Review by xeviknal — COMMENTED** [review]
bulk review

File: main.py (hunk 1 above)
Comment (xeviknal) [reply main.py]: no, tio. que esto lo peta. creo que si lo miras de otra manera seras mas feliz.

**Review by ziritrion — APPROVED** [review]
Ok, I like this

**Pull request merged**